kolibri/core/assets/src/views/app-bar.vue

```diff
@@ -75,28 +72,9 @@
               category="action"
             />
           </core-menu-option>
-          <core-menu-option
-            :label="$tr('signOut')"
-            @select="kolibriLogout"
-          />
+          <logout v-if="isUserLoggedIn" />
         </template>
 
-        <template v-else slot="options">
```

Comment:
No longer seeing sign in option. Intended behavior?

Reply:
No, not intended, thanks for checking, will double check.

Reply:
Ah, it's in the side nav, but not the app bar. Will fix!